packages/openfeature-node-provider/README.md

```diff
@@ -0,0 +1,114 @@
+# Bucket Node.js Provider
+
+This provider is an OpenFeature implementation for [Bucket](https://bucket.co) feature management service.
```

Comment:
Adapter rather

Reply:
what do you mean exactly?

Reply:
Sorry, was on the phone when writing so was short on details, I meant, in order to disambiguate with the actual "provider", we should call this library an "Adapter for Open Feature" instead.

Reply:
In OpenFeature nomenclature, this is called a "provider": https://github.com/open-feature/js-sdk-contrib/tree/main/libs/providers

Reply:
Ah, my bad